cpp/src/holtwinters/HoltWinters.cpp

```diff
@@ -0,0 +1,404 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cuda_runtime.h>
+
+#include <iostream>
+#include "HoltWinters.hpp"
+#include "holtwinters_utils.hpp"
+#include "hw_cu_utils.hpp"
+#include "utils.h"
+
+ML::HWStatus ML::HWInit() {
```

Comment:
Wrap these all in a `namespace ML { }` so that you don't need to use `ML::` everywhere.